4.0.0/migration.md

```diff
@@ -166,24 +298,29 @@ Stopping the MySQL migration job after 10 hours indicates that the `mysql-latest
         deploy.go:1583] waiting mysql latest migrations job to finish... (attempt 5/3600)
         deploy.go:1583] waiting mysql latest migrations job to finish... (attempt 6/3600)
         deploy.go:1583] waiting mysql latest migrations job to finish... (attempt 7/3600)
+        ...
+        deploy.go:1583] waiting mysql latest migrations job to finish... (attempt 3600/3600)
+        deploy.go:1587] mysql-latest-migrations-job failed
+
 
+1. It’s also safe to stop the Installer with `CTRL + c` instead of waiting for 10 hours: the migration will be retried on the next run, as it's an idempotent operation.
 
-1. Stop the installer  with CTRL + c because the migration will be retried on the next run.
-2. Check which initialization failed. To do so, describe the pod related to the `mysql-latest-migrations` job.
+2. Check which MySQL migration failed. To do so, describe the pod related to the `mysql-latest-migrations` job:
 
     `kubectl -n sysdigcloud describe pod -l "job-name=mysql-latest-migrations"`
 
 3. Find out which container in the pod is not terminated correctly by looking at `State` and `Reason` fields.
-   It should indicate that `scanning-migrations`, `draios-migrations` are the failed containers.
 
-4. Check the logs associated with the container that was failed to initialize:
+   It should indicate that `scanning-migrations` or `draios-migrations` are the failed containers.
+
+4. Check the logs associated with the container that has failed to complete:
 
     `kubectl -n sysdigcloud logs -l "job-name=mysql-latest-migrations" -c draios-migrations`
 
 
 ### MySQL To PostgreSQL Migration Job Failed
 
-Stopping the MySQL to PostgreGRE migration job after 10 hours indicates that the `mysql-latest-migrations` job has not finished correctly.
+The whole migration process will stop if after 10 hours the `mysql-to-postgres-migration` job has not finished correctly.
```

Comment:
The whole migration process might be terminated after 10 hours. This indicates that the `mysql-to-postgres-migration` job has not finished correctly.